data/qt-example/shader.frag

```diff
@@ -8,4 +8,5 @@ in vec4 color;
 void main()
 {
     fragColor = color;
+    fragColor = vec4(fragColor.x, 0.0, fragColor.y, 1.0);
```

Comment:
I suppose this is debug code to test the behavior?
The test is to toggle this line, save and reload while the application is running and see if the fragment colors change?